versions.yml

```diff
@@ -542,4 +542,4 @@ modGroups:
       homepage: https://modrinth.com/mod/notenoughcrashes
       modrinthId: yM94ont6
       desc: Sends you to title screen instead of crashing your game
-      url: https://cdn.modrinth.com/data/yM94ont6/versions/m0fQ0ZnK/notenoughcrashes-4.4.1%2B1.19.4-fabric.jar
\ No newline at end of file
+      url: https://cdn.modrinth.com/data/yM94ont6/versions/m0fQ0ZnK/notenoughcrashes-4.4.1%2B1.19.4-fabric.jar
```

Comment:
can you actually revert this change..?

Reply:
I'm not sure what was this change exactly. Probably something GitHub did automatically. It removed an empty line at the end of the file, or added it?

Reply:
well, github doesnt do anything automatically. Maybe it was your editor. If you remove the extra new line, it should be the same.

Reply:
Done in https://github.com/Wynntils/launchy/pull/38/commits/fe9d2d5cfe2af2508119234b32ca410c32bea370. For some reason, it still shows up :/

I'm editing right here in GitHub editor.

Can that empty line break something?